### PR TITLE
Change 'lookup key' terminology to 'label'

### PR DESCRIPTION
**Pull request by Bren2010**

### CODE REVIEW

File: draft-ietf-keytrans-architecture.md [hunk 1]
```diff
@@ -727,63 +727,64 @@ opening and the associated data, effectively anonymizing the entry.
...
 # Implementation Guidance
 
 Fundamentally, KT can be thought of as guaranteeing that all the users of a
-service agree on the contents of a key-value database. Using this guarantee,
-that all users agree on a set of keys and values, to authenticate the
+service agree on the contents of a key-value database (noting that this document
+refers to these keys as "labels"). Using this guarantee,
+that all users agree on a set of labels and values, to authenticate the
 relationship between end-user identities and the end-users of a communication
 service takes special care. Critically, in order to authenticate an end-user
```
Comment (felixlinker) [inline draft-ietf-keytrans-architecture.md:789]: I find this sentence very, very hard to parse.

**Review by felixlinker — APPROVED** [review]
I find one sentence very hard to parse, but it was before your changes too, so PR looks good to me!

**Review by thibmeu — APPROVED** [review]
aligns with what has been discussed in the interim. lgtm!
